open-metadata-resources/open-metadata-deployment/certificates/README.md

```diff
@@ -1,17 +1,43 @@
 <!-- SPDX-License-Identifier: CC-BY-4.0 -->
 <!-- Copyright Contributors to the Egeria project. -->
 
```

Comment:
This README is hidden - it requires knowledge of the project structure to find it.  It should be part of the Admin Guide (around configuring the platform).

Reply:
Thanks. 

Will move most content to
 https://github.com/odpi/egeria/blob/master/open-metadata-implementation/admin-services/docs/user/configuring-the-omag-server-platform.md with a link back to the directory originally in this PR just for the certificate creation artifacts